### PR TITLE
feat(quotes): add Turkish quotes from various sources (@turkwr)

### PR DESCRIPTION
**Pull request by turkwr**
### Description

New Turkish quotes added

### Checks

- [x] Adding quotes?
  - [x] Make sure to include translations for the quotes in the description (or another comment) so we can verify their content.
- [ ] Adding a language or a theme?
  - [ ] If is a language, did you edit `_list.json`, `_groups.json` and add `languages.json`?
  - [ ] If is a theme, did you add the theme.css?
    - Also please add a screenshot of the theme, it would be extra awesome if you do so!
- [ ] Check if any open issues are related to this PR; if so, be sure to tag them below.
- [x] Make sure the PR title follows the Conventional Commits standard. (https://www.conventionalcommits.org for more info)
- [x] Make sure to include your GitHub username inside parentheses at the end of the PR title

### CODE REVIEW

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/9897437265

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/9897474074

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/9897510985

File: frontend/static/quotes/turkish.json [hunk 1]
```diff
@@ -793,6 +793,36 @@
       "source": "Oscar Wilde",
       "length": 75,
       "id": 148
+    },
+    {
+      "text": "Bu hayatta büyük şeyler yapamayız. Sadece büyük bir sevgiyle küçük şeyler yapabiliriz.",
+      "source": "Mother Teresa",
+      "length": 79,
+      "id": 149
```
Comment (turkwr) [inline frontend/static/quotes/turkish.json:800]: We cannot do big things in this life. Only with great love can we do small things.

File: frontend/static/quotes/turkish.json [hunk 2]
```diff
@@ -793,6 +793,36 @@
...
+    },
+    {
+      "name": "Bir yaprak, sanki güneş ışınlarından destek alıyormuş gibi pencereden içeri süzüldü.",
```
Comment (turkwr) [inline frontend/static/quotes/turkish.json]: A leaf floated in through the window, as if supported by the sun's rays.

File: frontend/static/quotes/turkish.json [hunk 3]
```diff
@@ -793,6 +793,36 @@
...
+      "source": "Anais Nin",
+      "length": 78,
+      "id": 150
+    },
+    {
+      "name": "Umut, ruha tüneyen ve sözleri olmadan melodiyi söyleyen ve hiç durmayan tüylü bir şeydir.",
```
Comment (turkwr) [inline frontend/static/quotes/turkish.json]: Hope is a feathery thing that perches on the soul and sings the melody without words and never stops

File: frontend/static/quotes/turkish.json [hunk 4]
```diff
@@ -793,6 +793,36 @@
...
+      "source": "Emily Dickinson",
+      "length": 108,
+      "id": 151
+    },
+    {
+      "name": "Olasılıklar ne kadar saçma görünürse görünsün, içimizde bu zorlukların üstesinden gelme ve güzel bir şey başarma gücü vardır. Bir gün başladığımız yere dönüp baktığımızda ne kadar ilerlediğimizi görüp hayrete düşeceğiz.",
```
Comment (turkwr) [inline frontend/static/quotes/turkish.json]: No matter how absurd the odds may seem, we have the power within us to overcome these difficulties and achieve something beautiful. One day we will look back at where we started and marvel at how far we have come.

File: frontend/static/quotes/turkish.json [hunk 5]
```diff
@@ -793,6 +793,36 @@
...
+      "source": "Technoblade",
+      "length": 214,
+      "id": 152
+    },
+    {
+      "name": "Hizmet personeline davranış biçimlerine bakarak bir kişinin karakteri hakkında çok şey söyleyebilirsiniz.",
```
Comment (turkwr) [inline frontend/static/quotes/turkish.json]: You can tell a lot about a person's character by the way they treat their service staff.

File: frontend/static/quotes/turkish.json [hunk 6]
```diff
@@ -793,6 +793,36 @@
...
+      "source": "Bilinmeyen",
```
Comment (turkwr) [inline frontend/static/quotes/turkish.json:822]: Unkown

**Comment by Miodec** [comment]
Thanks

**Pull request merged**